src/Uno.UWP/UI/ColorOffset.cs

```diff
@@ -27,6 +27,8 @@ private ColorOffset(int a, int r, int g, int b)
 			B = b;
 		}
 
+		public static ColorOffset Zero = default;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'Zero' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)